### PR TITLE
Add README on Windows dGPU

### PR DESCRIPTION
**Pull request by JinBridger**

### CODE REVIEW

**Review by Oscilloscope98 — COMMENTED** [review]
Please rebase the description based on new readme :) especially for oneAPI related contents

File: README_windows_dgpu.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+set SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
+set SYCL_CACHE_PERSISTENT=1
+set BIGDL_QUANTIZE_KV_CACHE=1
+set BIGDL_LLM_XMX_DISABLED=1
+
+set no_proxy=localhost,127.0.0.1
+set BIGDL_IMPORT_IPEX=0
+
+python startup.py -a
+```
```
Comment (Oscilloscope98) [inline README_windows_dgpu.md:68]: Add note box here

>[!Note]
> The above configurations lead to optimal performance for **Intel Arc™ A-Series Graphics** with the exception of Intel Arc™ A300-Series or Pro A60.

```
>[!Note]
> The above configurations lead to optimal performance for **Intel Arc™ A-Series Graphics** with the exception of Intel Arc™ A300-Series or Pro A60.
```

File: README_windows_dgpu.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+## Installation
+
+### Download Langchain-Chatchat
+
+Download the Langchain-Chatchat with IPEX-LLM integrations from [this link](https://github.com/intel-analytics/Langchain-Chatchat/archive/refs/heads/ipex-llm.zip). Unzip the content into a directory, e.g. `C:\Users\arda\Downloads\Langchain-Chatchat-ipex-llm`. 
+
+### Install Prerequisites
+
+Visit the [Install IPEX-LLM on Windows with Intel GPU Guide](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Quickstart/install_windows_gpu.html), and follow [**Install Prerequisites**](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Quickstart/install_windows_gpu.html#install-prerequisites) to install Visual Studio, GPU driver, oneAPI, and Conda.
```
Comment (Oscilloscope98) [inline README_windows_dgpu.md]: Visit the [Install IPEX-LLM on Windows with Intel GPU Guide](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Quickstart/install_windows_gpu.html), and follow [**Install Prerequisites**](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Quickstart/install_windows_gpu.html#install-prerequisites) to install [Visual Studio](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Quickstart/install_windows_gpu.html#install-visual-studio-2022), [GPU driver](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Quickstart/install_windows_gpu.html#install-gpu-driver), and [Conda](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Quickstart/install_windows_gpu.html#install-miniconda).

**Comment by Oscilloscope98** [comment]
@shane-huang

Here are the draft setup guide for Windows dGPU users :)

We will add tips for select device when having multiple GPUs, disable device, etc. later

**Review by shane-huang — APPROVED** [review]
LGTM.

**Pull request merged**